tests/package.json

```diff
@@ -30,5 +30,8 @@
   },
   "dependencies": {
     "@jahia/jahia-reporter": "^1.0.3"
+  },
+  "resolutions": {
+    "minimist" : "^1.2.6"
```

Comment:
we can probably remove this already since minimist dependencies should already resolve to 1.2.6